models/devices/gazebo/ray.rb

```diff
@@ -10,6 +11,7 @@ module Gazebo
             device_type "Ray" do
                 provides Entity
                 provides CommonModels::Services::LaserScan
+                provides Seabots::Services::DepthMap
```

Comment:
Create a DepthMap service in common models. This is an open source repo and Seabots is a private one

Reply:
Done

Reply:
Done @jhonasiv